streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -643,7 +643,7 @@ public void shouldAddInternalTopicConfigForRepartitionTopics() {
         final InternalTopologyBuilder.TopicsInfo topicsInfo = builder.topicGroups().values().iterator().next();
         final InternalTopicConfig topicConfig = topicsInfo.repartitionSourceTopics.get("appId-foo");
         final Map<String, String> properties = topicConfig.getProperties(Collections.<String, String>emptyMap(), 10000);
-        assertEquals(5, properties.size());
+        assertEquals(3, properties.size());
         assertEquals(String.valueOf(Long.MAX_VALUE), properties.get(TopicConfig.RETENTION_MS_CONFIG));
```

Comment:
I think this needs to be changed to -1 to fit with what is being set in the properties. At the moment this test fails consistently.

Reply:
@soenkeliebau That was my bad, I ran than unit test while this change was stashed. Thanks!